autode/mol_graphs.py

```diff
@@ -158,8 +155,10 @@ def set_graph_attributes(species):
     logger.info('Setting the π bonds in a species')
 
     def is_idx_pi_atom(idx):
-        return is_pi_atom(atom_label=species.atoms[idx].label,
-                          valency=species.graph.degree[idx])
+        if type(species.atoms[idx].is_pi) is bool:
```

Comment:
I hate this but I couldn't think of a better solution

Reply:
whats wrong with just `return species.atoms[idx].is_pi(valency=species.graph.degree[idx])`

Reply:
because with a `SMILESAtom.is_pi` is an attribute 😕. Defining is_pi as a method that ignores any arguments for `SMILESAtom` is best?